zipline/examples/buy_and_hold.py

```diff
@@ -33,18 +30,12 @@ def handle_data(context, data):
         context.has_ordered = True
 
 
-if __name__ == '__main__':
+def _test_args():
+    """Extra arguments to use when testing this example.
```

Comment:
What do you think of "Extra arguments to use when zipline automated tests run this example."?